Jenkinsfile

```diff
@@ -189,11 +189,7 @@ stage('Publish') {
                     """
                     sh "scripts/create-github-release.js"
 
-                    // Publish the updated package to the npm registry.
-                    // Use `npm` rather than `yarn` for publishing.
-                    // See https://github.com/yarnpkg/yarn/pull/3391.
-                    sh "echo '//registry.npmjs.org/:_authToken=${env.NPM_TOKEN}' >> \$HOME/.npmrc"
-                    sh "npm publish --tag ${npmTag}"
+                    sh "ENV_TOKEN=${env.NPM_TOKEN} yarn publish --no-interactive --tag ${npmTag} --new-version=${newPkgVersion}"
```

Comment:
Where does `ENV_TOKEN` come from? I couldn't find a reference to that environment variable as something that yarn would recognize via Google.

Reply:
Sorry, my bad. I correct it to `NPM_TOKEN`.

Alternatively, the previous line could be kept:
`sh "echo '//registry.npmjs.org/:_authToken=${env.NPM_TOKEN}' >> \$HOME/.npmrc"`